src/main/java/org/isf/dlvrrestype/model/DeliveryResultType.java

```diff
@@ -21,16 +21,29 @@
  */
 package org.isf.dlvrrestype.model;
 
+import java.io.Serializable;
+
+import org.isf.utils.db.Auditable;
```

Comment:
The imports are out of order; `org` should be last.
See https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/eclipse/OpenHospital.importorder